internal-gateway/deployment.yaml

```diff
@@ -36,25 +36,50 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       containers:
        - name: internal-gateway
-         image: "{{ internal_gateway_image.image }}"
+         image: "{{ global.docker_prefix }}/envoy:v1.22.3"
+         command:
+           - /usr/local/bin/envoy
+           - --config-path
+           - /etc/envoy/envoy.yaml
+           - --concurrency
+           - "2"
+           - --service-cluster
+           - gateway
```

Comment:
I think this should be` internal-gateway`

Reply:
Good catch